lib/confusables/generate_list.go

```diff
@@ -24,6 +25,11 @@ package %s
 `
 )
 
+var mishapsReplacer = strings.NewReplacer(
+	"vv", "w",
+	"rn", "m",
+)
+
```

Comment:
It might be beneficial to move this closer to `FixIssuesWithStr` and add a brief comment.

Reply:
Alright